cmd/gateway-main.go

```diff
@@ -112,9 +112,16 @@ func StartGateway(ctx *cli.Context, gw Gateway) {
 		cli.ShowCommandHelpAndExit(ctx, gatewayName, 1)
 	}
 
+	// Get "json" flag from command line argument.
+	jsonFlag := ctx.Bool("json") || ctx.GlobalBool("json")
+	if jsonFlag {
+		log.EnableJSON()
```

Comment:
We can have `EnableQuiet ` here if jsonFlag is turned on.
I think this way, it'll be easier to understand and complete instead of having `jsonFlag` checked one more time at line#124 together with `quietFlag`.
```
	if jsonFlag {
 		log.EnableJSON()
	}
```
 =>
```
	if jsonFlag {
 		log.EnableJSON()
 		log.EnableQuiet()
	}
```

Reply:
done